src/WebJobs.Script/Utility.cs

```diff
@@ -792,6 +793,28 @@ public static bool IsMediaTypeOctetOrMultipart(MediaTypeHeaderValue mediaType)
             }
         }
 
+        public static void LogAutorestGeneratedJson(string rootScriptPath, ILogger logger = null)
+        {
```

Comment:
do not default to `null` - this will throw null ref if accidentally called without passing logger.